### PR TITLE
Recorder

### PR DESCRIPTION
**Pull request by frostbyte73**

### CODE REVIEW

File: recorder-sample.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    # password:
+  # api key+secret required if recording with templates by room name only
+  apiKey: "server-api-key"
+  apiSecret: "server-api-secret"
+  # default input options
+  input:
+    width: 1920
+    height: 1080
+    depth: 24
+    framerate: 25
```
Comment (davidzhao) [inline recorder-sample.yaml:19]: why do we default to fps of 25 instead of 30?

**Review by davidzhao — APPROVED** [review]
lgtm! one Q about fps.

File: recorder-sample.yaml (hunk 1 above)
Comment (frostbyte73) [reply recorder-sample.yaml:19]: 24/25 is the minimum for smooth looking video

Comment (frostbyte73) [reply recorder-sample.yaml:19]: we can try recording with both and see if there's a noticeable difference

Comment (davidzhao) [reply recorder-sample.yaml:19]: I think 30fps is commonly used due to US video standard. I've rarely seen 24 for online video
0E8FFBB8-FBAA-4AB2-A29C-822CB5A83914

**Pull request merged**